help_to_heat/frontdoor/views.py

```diff
@@ -660,6 +689,9 @@ def build_extra_context(self, request, session_id, page_name, data, is_change_pa
         return {
             "uprn_options": uprn_options,
             "manual_url": page_name_to_url(session_id, address_select_manual_page, is_change_page),
+            "current_month_start": current_month_start,
+            "next_month_start": next_month_start,
+            "country": country,
```

Comment:
probably better to send a show/hide flag rather than the country to take logic out of the template

Reply:
yeah, I've made this into a flag instead